samples/SMPElistDefaults.yaml

```diff
@@ -0,0 +1,12 @@
+SMPECSI:
+  Dataset: AS4SMP.GLOBAL.CSI
```

Comment:
This should follow the naming conventions of the surrounding code. In this case python so `Dataset` should be `dataset`.

This goes for the other variables as well. I'm assuming the HLQs are that way to be synonymous with ZOS but they should also follow that convention if not.